README.md

```
@@ -117,6 +119,12 @@ To deploy the Smart Desk Server on your microcomputer and set up remote developm
    $ rm ~/.zshrc
    $ ln -s ~/open-smart-desk/conf/home/ubuntu/.zshrc ~/.zshrc
    ```
+* Install project dependencies and build:
+   ``` bash
+   $ cd ~/open-smart-desk/server
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)